lib/install/management/create.go

```diff
@@ -220,31 +220,7 @@ func (d *Dispatcher) cleanupAfterCreationFailed(conf *config.VirtualContainerHos
 			d.op.Debug("Successfully cleaned up dangling bridge network.")
 		}
 	}
-
-	// cleanup the vch inventory folder if it is empty. Can happen in some cases where the create is cancelled after successful creation.
-
-	if d.isVC {
-		folderRef, err := vchFolder(d.op, d.session, conf)
-		if folderRef != nil {
-			children, err := folderRef.Children(d.op)
-			if err != nil {
-				d.op.Debugf("encountered error during vch folder cleanup : %s", err)
-				d.op.Warnf(manualInventoryCleanWarning, folderRef.InventoryPath)
-			}
-
-			if len(children) != 0 {
-				d.op.Warnf(manualInventoryCleanWarning, folderRef.InventoryPath)
-			}
-
-			d.removeFolder(folderRef)
-			if err != nil {
-				d.op.Warnf(manualInventoryCleanWarning, folderRef.InventoryPath)
-			}
-		}
-		if err != nil {
-			d.op.Debugf("encountered error during vch folder cleanup : %s", err)
-		}
-	}
+	d.deleteVCHFolder()
```

Comment:
I like this, originally we had removed my `deleteVCHFolder` function and I felt that I was repeating code and that it could have been better. thank you.